examples/blog-with-comment/README.md

```diff
@@ -1,65 +1,72 @@
 # Blog with Comment
 
-This project adds commenting functionality to [Next.js blog application](https://github.com/vercel/next.js/tree/canary/examples/blog) using Upstash and Auth0.
+This project adds commenting functionality
+to [Next.js blog application](https://github.com/vercel/next.js/tree/canary/examples/blog)
+using Upstash and NextAuth.
 
-The comment box requires Auth0 authentication for users to add new comments. A user can delete their own comment. Also admin user can delete any comment.
+The comment box requires Auth0 authentication for users to add new comments. A
+user can delete their own comment. Also admin user can delete any comment.
 
 Comments are stored in Serverless Redis ([Upstash](http://upstash.com/)).
 
 ### Demo
 
 [https://blog-with-comment.vercel.app/](https://blog-with-comment.vercel.app/)
 
-## `1` Project set up
+## `1` Set up project
```

Comment:
For this section, let's follow other example readmes like: https://github.com/vercel/next.js/tree/canary/examples/hello-world#how-to-use